packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/server.ts

```diff
@@ -53,7 +53,7 @@ export function getServerConfig(wco: WebpackConfigOptions): Configuration {
       ...externalDependencies,
       (context: string, request: string, callback: (error?: null, result?: string) => void) => {
         // Absolute & Relative paths are not externals
-        if (request.startsWith('./') || isAbsolute(request)) {
+        if (request.startsWith('./') || request.startsWith('../') || isAbsolute(request)) {
```

Comment:
Can be simplified to `request.startsWith('.')`

Reply:
I personally find the intention expressed clearer the way it is. But I'm ok with the change if it better matches the codebase.

Reply:
On a second thought - wouldn't it make the condition too permissive?

Reply:
External modules cannot start with dot, since that’s an invalid name. Therefore, all paths that start with a dot must be relative.

Reply:
Also, in other classes we are using the same pattern.